test/sql/go.mod

```diff
@@ -5,17 +5,16 @@ go 1.23.0
 toolchain go1.23.8
 
 require (
-	github.com/cloudevents/sdk-go/sql/v2 v2.14.0
-	github.com/cloudevents/sdk-go/v2 v2.15.2
+	github.com/cloudevents/sdk-go/sql/v2 v2.15.2
```

Comment:
@duglin FYI, either there was a breaking change or there's some other reason why I couldn't force the go toolchain to upgrade to `v2.16.0` here.

Reply:
ignore, fixed due to missing replace - man...that codebase :)